terra-node-dev/test/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Is this package.json used to run tests with docker-compose? or is this guy still needed?

Reply:
it's used for tests